lib.sh

```diff
@@ -313,15 +313,19 @@ explainer_popup() {
 }
 
 reinstall_remove_menu() {
-    choice=$(whiptail --title "$TITLE" --menu "It seems like '$SCRIPT_NAME' is already installed.\nChoose what you want to do." "$WT_HEIGHT" "$WT_WIDTH" 4 \
+    choice=$(whiptail --title "$TITLE" --menu "It seems like $SCRIPT_NAME is already installed.\nChoose what you want to do." "$WT_HEIGHT" "$WT_WIDTH" 4 \
     "Reinstall $SCRIPT_NAME" "" \
     "Uninstall $SCRIPT_NAME" "" 3>&1 1>&2 2>&3)
+    if [ -z "$choice" ]
```

Comment:
We should add this everywhere. Isn't this exiting when you hit cancel? Currently that's not working.

Reply:
done